POSTINSTALL.md

```diff
@@ -29,11 +22,8 @@ Add new document to FireStore messages collection and we will deliver it for you
   }
 }
 ```
-
 **Note:** Different documents can be delivered on different channels by changing the channel IDs. You can get the channel ID from your [Channels page](https://dashboard.messagebird.com/en/channels/)
-
```

Comment:
Same here, let's keep newlines in place? I do not think they asked for removing them